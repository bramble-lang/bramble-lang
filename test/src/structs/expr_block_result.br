fn my_main() {
    let t: MyStruct := if (true) {
        let x:i64 := 5;
        let y:i64 := 1;
        MyStruct{x: x, y: y, z: true}
    } else {
        MyStruct{x: 0, y: -1, z: false}
    };
    printiln t.x;
    printiln t.y;
    printbln t.z;
    
    return;
}

struct MyStruct {
    x: i64,
    y: i64,
    z: bool,
}