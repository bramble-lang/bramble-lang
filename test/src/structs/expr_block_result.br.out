5
1
true
